docs/config.rst

```diff
@@ -102,6 +102,9 @@ examples first and use this page as a reference.
 Core
 ----
 
+The following options are set in the ``[tox]`` section of ``tox.ini`` or
```

Comment:
Can you please align to 120 characters?

Reply:
Done.

This is not standard for Python, your linter doesn't check for it, but the note about this is a bit buried in your development docs. Do you want a PR to bring this a little more to the fore so that new developers don't miss it?

Reply:
What do you have in mind?

Reply:
As a starting point for discussion, I've put an initial go at changes on my `dev/cjs/24a20/doc-config-mechanisms` branch. I am far from sure I understand yet what you're saying about configuration mechanisms, but this is what I've got so far.

Reply:
For a start, it should be moved up to the top of the "style guide" section, since your linter does not flag standard Python style as wrong. I'll think about it further after dealing with some of the other docs stuff.